config/config.toml

```diff
@@ -17,7 +17,7 @@
 #
 # Eg 3:
 # [Gateway]
-# maxDBWriterProcess = 64
+# maxUserWebRequestWorkerProcess = 64
```

Comment:
Leave this as it is? That variable is still present

Reply:
Done